app/assets/stylesheets/common/base/share_link.scss

```diff
@@ -38,6 +38,13 @@
     display: flex;
     justify-content: flex-end;
     margin-left: auto;
+    align-items: baseline;
+
+    .date {
+      font-weight: normal;
+      color: var(--primary-med-or-secondary-med);
+      margin-right: 0.5em;
```

Comment:
Nitpick, but we only need the margin when the button is shown, so it's best to add a `.date + .btn { margin-left: 0.5em;}` rule instead.

Reply:
I moved the new topic button to the left, so the margin-right is no longer necessary. 👍🏻